content/bn/style-guide/_index.md

```diff
@@ -7,125 +7,121 @@ menu:
+title: সংজ্ঞা টেমপ্লেট
 status: Feedback Appreciated
-category: Concept
+category: ধারণা
 ---
-```
 
-### Definition
+### সংজ্ঞা
+
+#### তিনটি উপশিরোনাম
+
+*প্রযুক্তি* এবং *ধারণা* বিভাগের সংজ্ঞায় তিনটি উপশিরোনাম রয়েছে:
 
-#### Three subheadings
+- *এটা কি*: আমরা যে বিষয়ে কথা বলছি তার একটি সংক্ষিপ্ত এবং স্পষ্ট ধারণা প্রদান করুন।
+- 
+- *এটা যেসব সমস্যাতে ফোকাস করে*: সমস্যাটির উপর ফোকাস করুন, সমাধানের দিকে নয় (যা পরবর্তী বিভাগে আসে)। আসলে, সংজ্ঞায়িত শব্দটি উল্লেখ করা এড়িয়ে চলুন। সমস্যাটি আমাদের সেই জিনিসটির প্রয়োজন কিসের উপর তাতে আলোকপাত করে ।
+- 
+- *এটা কিভাবে সাহায্য করে*: এখন, মেয়াদে ফিরে আসুন।  এটি কিভাবে উপরে বর্ণিত সমস্যার সমাধান করে?
 
-The definitions for **technology** and **concept** categories contain three subheadings: 
+মনে রাখবেন **বৈশিষ্ট্য**-এর আলাদা বিভাগের প্রয়োজন নেই। একটি সংজ্ঞা যথেষ্ট হবে।
 
-- **What it is**: provide a short and clear overview of what we are talking about.
-- **Problem it addresses**: focus on the problem, not the solution (that comes in the next section). In fact, avoid mentioning the term that is defined. The problem focuses on *what* led us to need that thing. 
-- **How it helps**: now, come back to the term. How does it address the problem described above?
+#### সহজ রাখুন
 
-Note that **properties** don't require separate sections. A definition will suffice. 
+শব্দকোষের লক্ষ্য *জটিল ধারণাগুলিকে সহজ শব্দে ব্যাখ্যা করা* — এটি একটি আশ্চর্যজনকভাবে কঠিন কাজ যা সম্ভবত একাধিক সংশোধন করতে পারে। আপনার সংজ্ঞা খসড়া করার সময় সর্বদা দর্শকদের কথা মাথায় রাখুন। শিল্পের শর্তাবলী এবং বাজওয়ার্ডগুলি ব্যবহার করা এড়িয়ে চলুন - আপনি সম্ভবত তাদের কাছে ফিরে যেতে পারবেন এবং স্বয়ংক্রিয়ভাবে সংশোধন করতে হবে।
 
-#### Keeping it simple
+উপযুক্ত হলে, *বাস্তব-জগতের উদাহরণ* ব্যবহার করুন যা পাঠকদের (বিশেষ করে অ-প্রযুক্তিগত) আরও ভালোভাবে বুঝতে সাহায্য করে কখন এবং কেন আপনি যে ধারণাটি ব্যাখ্যা করছেন তা প্রাসঙ্গিক।
 
-The Glossary aims at **explaining complex concepts in simple words** — that is a surprisingly difficult task that will likely take multiple revisions. Always keep the audience in mind when drafting your definition. Avoid using industry terms and buzzwords — you'll probably catch yourself going back to them and may need to autocorrect. 
+আপনার সংজ্ঞায় ব্যবহৃত হলে, সর্বদা *বিদ্যমান শব্দকোষের শর্তাবলীর সাথে লিঙ্ক করুন* (শুধুমাত্র প্রথম উল্লেখ হাইপারলিঙ্ক করা উচিত)।
 
-When appropriate, use **real-world examples** that help readers (especially non-technical ones) better understand *when* and *why* the concept you’re explaining is relevant. 
+*উদাহরণ*: [পরিষেবা মেশ সংজ্ঞা](https://glossary.cncf.io/service_mesh/) এর “এটি কী” বিভাগটি একবার দেখুন। এটি মাইক্রোসার্ভিস, পরিষেবা, নির্ভরযোগ্যতা এবং পর্যবেক্ষণযোগ্যতার সংজ্ঞাগুলির সাথে লিঙ্ক করে। উপরন্তু, এটি একটি মাইক্রোসার্ভিসেস পরিবেশে নেটওয়ার্ক চ্যালেঞ্জের তুলনা করে একটি বাস্তব-বিশ্বের উদাহরণ ব্যবহার করে (এমন কিছু যা অ-প্রযুক্তিগত লোকেরা সম্পর্কিত হতে পারে না)  ওয়াইফাই সমস্যার (যা কেউ ল্যাপটপ ব্যবহার করে বুঝতে পারে)সাথে । যেখানে সম্ভব, সেই সংযোগটি তৈরি করার চেষ্টা করুন।
```

Comment:
From https://github.com/cncf/glossary/pull/740:

I recommend to remove hardcoded site domain name:

Replace `https://glossary.cncf.io/service_mesh/` → `/service_mesh/`